CMakeLists.txt

```diff
@@ -7,8 +7,8 @@ disallow_intree_builds()
 project (utf8proc C)
 
 # Be sure to also update these in Makefile!
-set(SO_MAJOR 3)
-set(SO_MINOR 0)
+set(SO_MAJOR 2)
```

Comment:
In particular, because of the changes to the `utf8proc_property_struct` data structure.   Or do we not consider those fields part of the public API?